drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/pom.xml

```diff
@@ -130,7 +130,11 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
-    <!-- END TEST -->
+      <dependency>
+          <groupId>org.uberfire</groupId>
+          <artifactId>appformer-kogito-bridge</artifactId>
```

Comment:
Scope test?
If not it is in the wrong dependency section :)

Reply:
@danielezonca Wrong section, moved